### PR TITLE
Updates to headless chrome article

### PR DESCRIPTION
**Pull request by ebidel**
R: @petele 

Switch to `async/await` to make examples easier to read.

### CODE REVIEW

File: src/content/en/updates/2017/04/headless-chrome.md [hunk 1]
```diff
@@ -171,34 +181,26 @@ use it, first install Lighthouse from npm:
...
 ```javascript
-const {ChromeLauncher} = require('lighthouse/lighthouse-cli/chrome-launcher');
+const chromeLauncher = require('lighthouse/chrome-launcher/chrome-launcher');
 
 /**
- * Launches a debugging instance of Chrome on port 9222.
+ * Launches a debugging instance of Chrome.
  * @param {boolean=} headless True (default) to launch Chrome in headless mode.
- *     Set to false to launch Chrome normally.
+ *     False launches a full version of Chrome.
```
Comment (jpmedley) [inline src/content/en/updates/2017/04/headless-chrome.md]: True and false conditions need to be described using the same sentence structure.

File: src/content/en/updates/2017/04/headless-chrome.md [hunk 2]
```diff
@@ -33,8 +33,7 @@ don't need a visible UI shell. For example, you may want to run some tests again
 a real web page, create a PDF of it, or just inspect how the browser renders an URL.
 
 Caution: Headless mode is available on Mac and Linux in **Chrome 59**.
-[Windows support](https://bugs.chromium.org/p/chromium/issues/detail?id=686608) is coming soon! To
-check what version of Chrome you have, open `chrome://version`.
+[Windows support](https://bugs.chromium.org/p/chromium/issues/detail?id=686608) is coming in Chrome 60. To check what version of Chrome you have, open `chrome://version`.
```
Comment (petele) [inline src/content/en/updates/2017/04/headless-chrome.md]: line exceeds 80 chars

File: src/content/en/updates/2017/04/headless-chrome.md (hunk 1 above)
Comment (petele) [inline src/content/en/updates/2017/04/headless-chrome.md]: @jpmedley - looks like the same to me, what would you suggest?

Comment (jpmedley) [inline src/content/en/updates/2017/04/headless-chrome.md]: True uses an infinitive form ("True to launch"). False uses a simple verb phrase ("False launches"). They need to be the same.

Comment (petele) [inline src/content/en/updates/2017/04/headless-chrome.md]: My English teachers would hang their head in shame. Can you please give a specific example of how they should both look?

Comment (jpmedley) [inline src/content/en/updates/2017/04/headless-chrome.md]: I would do this:

True (default) launches Chrome in headless mode.
False launches a full version of Chrome.

**Comment by ebidel** [comment]
PTAL

**Pull request merged**